messaging/kafka/client/consumer.go

```diff
@@ -311,6 +323,7 @@ func (ref *Consumer) errorHandler(in <-chan error) {
 	for {
 		select {
 		case err, more := <-in:
+			ref.Warn("ERROR")
```

Comment:
Either use more descriptive message or remove the log

Reply:
removed